Sources/StreamChatSwiftUI/ChatChannelList/ChatChannelListViewModel.swift

```diff
@@ -393,10 +402,11 @@ open class ChatChannelListViewModel: ObservableObject, ChatChannelListController
     /// Creates a new message search controller, sets its delegate, and triggers the search operation.
     open func performMessageSearch() {
         messageSearchController = chatClient.messageSearchController()
-        messageSearchController?.delegate = self
```

Comment:
Moved it because otherwise delegate makes a callback with 0 results before sync finishes and this makes the UI to flash.